catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/FederationAdminServiceImpl.java

```diff
@@ -437,6 +443,10 @@ public void deleteRegistryEntriesByMetacardIds(List<String> metacardIds,
                 .collect(Collectors.toList());
         List<Filter> filters = getBasicFilter();
         Filter filter = filterBuilder.allOf(filters);
+        if (includeInternal) {
+            filter = filterBuilder.anyOf(filter,
```

Comment:
We don't expect to have any more internal representations. One is confusing enough but needed to handle concurrency and multiple metacard sources.